vtex/faststore
# Feat: Adds `RegionModal` component

## What's the purpose of this pull request?

`fs/components`
- [x] Renames and moves `RegionModal` component
- [x] Adds `RegionModal ` doc page

`fs/ui`
- [x] Creates `RegionModal` styles

`fs/core`
- [x] Review `RegionModal` structure (Removes `RegionInput`)
- [x] Delete  old `Region`'s  related files (storybook and style)


## How to test it?

run `yarn dev` on `core` application and check the `RegionModal` feature on the homepage (Click on the `RegionBar` - Set your location button) or test it using the doc preview link:
- [RegionModal](https://faststore-site-git-feat-region-modal-fs-810-faststore.vercel.app/components/organisms/region-modal)


### Starters Deploy Preview

[WIP]
